plugins/node/opentelemetry-instrumentation-pg/test/pg.test.ts

```diff
@@ -259,6 +259,16 @@ describe('pg', () => {
       assert.strictEqual(res, undefined, 'No promise is returned');
     });
 
+    it('should pass the client connection object in the callback function', done => {
+      connClient.connect(function (err: Error, conn?: pg.Client) {
+        //the 'connect' function passes the client object as an argument
+        //in the callback function: https://github.com/brianc/node-postgres/blob/master/packages/pg/lib/client.js#L282
+        assert(arguments[1] === connClient);
+        assert(arguments[1] === conn);
```

Comment:
Thanks, I thought about this - making sure that connClient === conn is like verifying postgres works. because they send the "this" (aka the client) as a parameter to the function. So I left only the first assertion.

Reply:
If I assert this: `assert(conn === connClient)` this is like asserting what postgres does (here: https://github.com/brianc/node-postgres/blob/master/packages/pg/lib/client.js#L290). What I want to check, is if **our patched function** passes the client also. This is why I check `arguments[1] === connClient`.